rapidsms/contrib/registration/tests.py

```diff
@@ -195,7 +194,7 @@ def test_contact_update_add_connection(self):
             retval = views.contact(request, pk=contact.pk)
         self.assertTrue(isinstance(retval, HttpResponseRedirect))
         self.assertEqual(302, retval.status_code)
-        render.assert_called()
+        render.assert_not_called()
```

Comment:
The pyupgrade pre-commit hook replaced imports from the `mock` PyPI package with the built-in `unittest.mock`. We were using a very old version of `mock` (1.0.1, released in 2012). For some reason, the `assert_called()` was passing with that version of mock, which is odd because the [view under test](https://github.com/rapidsms/rapidsms/blob/f73b23d14d8b869e160f9039542e429e6253ebca/rapidsms/contrib/registration/views.py#L45-L88) is returning a redirect response (as evidenced by the two assertions immediately above this one), so we should not expect the `render` function to be called. With the change to `unittest.mock`, the `assert_called()` correctly fails, so I've changed it to `assert_not_called()`. FWIW, the `assert_called()` also fails with mock > 1.0.1 (actually raises an AttributeError in mock < 2.0.0).